ports/winsparkle/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "winsparkle",
+  "version-string": "0.7.0",

```

```suggestion
  "version": "0.7.0",
```
version seems more proper here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md